client/man/ipa-client-samba.1

```diff
@@ -0,0 +1,88 @@
+.\" A man page for ipa-client-samba
+.\" Copyright (C) 2008-2016  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-client-samba" "1" "Jun 10 2019" "FreeIPA" "FreeIPA Manual Pages"
+.SH "NAME"
+ipa\-client\-samba \- Configure Samba file server on an IPA client
+.SH "SYNOPSIS"
+ipa\-client\-samba [\fIOPTION\fR]...
+.SH "DESCRIPTION"
+Configures a Samba file server on the client machine to use IPA domain controller for authentication and identity services.
+
+The tool configures Samba file server to be a domain member of IPA domain. Samba file server will use SSSD to resolve information about users and groups, and will use IPA master it is enrolled against as its domain controller.
+
+It is not possible to reconciliate original Samba environment if that was pre-existing on the client with new configuration. Samba databases will be updated to follow IPA domain details and \fBsmb.conf\fR configuration will will be overwritten. It is recommended to enable Samba suite on a freshly deployed IPA client.
+
+.TP
+During the configuration process, the tool will perform following steps:
+
+1. Discover details of IPA domain: realm, domain SID, domain ID range
+
+2. Discover details of trusted Actvide Directory domains: domain name, domain SID, domain ID range
+
+3. Create Samba configuration file using the details discovered above.
+
+4. Create Samba Kerberos service using host credentials and fetch its keytab into /etc/samba/samba.keytab. The Kerberos service key is pre-set to a randomly generated value that is shared with Samba.
+
+5. Populate Samba databases by setting the domain details and the randomly generated machine account password from the previous step.
+
+6. Create a default [homes] share to allow users to log in to their home directories if \-\-no\-mkhomedir option was not specified.
```

Comment:
I think it would read better as:

...to their home directories unless the \-\-no\-homes option was specified.

Either way the option name needs to be fixed.